frontend/src/components/Map/MapContainer.js

```diff
@@ -1,11 +1,12 @@
 import React, {useState} from 'react';
-import {Map, GoogleApiWrapper, InfoWindow, Marker} from 'google-maps-react';
+import {Map, GoogleApiWrapper, InfoWindow, Marker, Polyline} from 'google-maps-react';
+import MapStyles from "./MapStyles";
```

Comment:
This should be lowercase (camelCase), since it's not a component :)

Reply:
got it, thanks :)